Dockerfile.builder

```diff
@@ -11,7 +11,7 @@ RUN go get -u github.com/golang/lint/golint
 RUN go get github.com/jstemmer/go-junit-report
 
 ### Install NodeJS and yarn
-ENV NODE_VERSION="v10.3.0"
+ENV NODE_VERSION="v10.17.0"
```

Comment:
Not a big deal, but noting that `yarn` latest is `1.21`.  Not sure if we want to bump these together.

Reply:
yarn is a bit different as we use the version checked into the repository, see #3667